Harmony/Public/Patching/PatchManager.cs

```diff
@@ -143,7 +143,7 @@ internal static MethodBase FindReplacement(StackFrame frame)
 
 			lock (ReplacementToOriginals)
 				return ReplacementToOriginals
-					.FirstOrDefault(kv => kv.Key.IsAlive && ((MethodBase)kv.Key.Target).GetNativeStart().ToInt64() == methodStart).Key.Target as MethodBase;
+					.FirstOrDefault(kv => kv.Key.IsAlive && PlatformTriple.Current.Runtime.GetMethodEntryPoint((MethodBase)kv.Key.Target).ToInt64() == methodStart).Key.Target as MethodBase;
```

Comment:
This should almost certainly be using the PlatformTriple method rather than the Runtime method.